sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -824,6 +825,14 @@ public Response<BinaryData> getAudioTranslationWithResponse(
                 deploymentOrModelName, audioTranslationOptions, requestOptions);
     }
 
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getAudioTranslationWithResponse(
+        String deploymentOrModelName, BinaryData audioTranslationOptions, RequestOptions requestOptions, String boundary, String contentLength) {
```

Comment:
Do parameters `boundary` and `contentLength` are more frequently used than `requestOptions`?

Might be better to consider re-order the parameters:
`getAudioTranslationWithResponse(String deploymentOrModelName,  String boundary, String contentLength, BinaryData audioTranslationOptions, RequestOptions requestOptions) `

Reply:
there is unfortunately no support for multipart data at a protocol level from azure core that I am aware of. I am just attempting a rudimentary/explanatory hack to see if I can somehow reach the service with an interim implementation.